youtrack_mcp/config.py

```
@@ -56,9 +57,17 @@ def validate(cls) -> None:
         Raises:
             ValueError: If required settings are missing or invalid
         """
+        # Try to read token from file if no token is set but file is specified
+        if not cls.YOUTRACK_API_TOKEN and cls.YOUTRACK_TOKEN_FILE:
+            try:
+                with open(cls.YOUTRACK_TOKEN_FILE, 'r') as f:
+                    cls.YOUTRACK_API_TOKEN = f.read().strip()
+            except Exception as e:
+                raise ValueError(f"Failed to read YouTrack API token from file {cls.YOUTRACK_TOKEN_FILE}: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)